internal/hardware/validator.go

```diff
@@ -201,6 +199,30 @@ func totalizeRequirements(ocpRequirements models.ClusterHostRequirementsDetails,
 	return total
 }
 
-func (v *validator) GetInstallationDiskSpeedThresholdMs() int64 {
-	return v.InstallationDiskSpeedThresholdMs
+func (v *validator) getOCPHostRoleRequirementsForVersion(role models.HostRole, openShiftVersion string) models.ClusterHostRequirementsDetails {
+	requirements := v.getOCPRequirementsForVersion(openShiftVersion)
```

Comment:
We may need per version requirements for olm operators like ocs and cnv in the future.

Reply:
Agreed. In the future that might be needed like versioned requirements for OCP.